patches/chrome-android-java-src-org-chromium-chrome-browser-homepage-HomepageManager.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/homepage/HomepageManager.java b/chrome/android/java/src/org/chromium/chrome/browser/homepage/HomepageManager.java
+index de101add328a69dd406b9ce34881f76feb21ae23..c58148164d7f7dc931154cfd55d55ba053a4becd 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/homepage/HomepageManager.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/homepage/HomepageManager.java
+@@ -50,7 +50,7 @@ public class HomepageManager
+     private final ObserverList<HomepageStateListener> mHomepageStateListeners;
+     private SettingsLauncher mSettingsLauncher;
+ 
+-    private HomepageManager() {
```

Comment:
we can't make this public with asm?

Reply:
we can, but we need to have it public during compile time to subclass with [BraveHomepageManager](https://github.com/brave/brave-core/pull/23233/files?file-filters%5B%5D=.chromium&file-filters%5B%5D=.flags&file-filters%5B%5D=.gn&file-filters%5B%5D=.gni&file-filters%5B%5D=.grd&file-filters%5B%5D=.icon&file-filters%5B%5D=.java&file-filters%5B%5D=.js&file-filters%5B%5D=.json5&file-filters%5B%5D=.lock&file-filters%5B%5D=.mm&file-filters%5B%5D=.py&file-filters%5B%5D=.toml&file-filters%5B%5D=.ts&file-filters%5B%5D=No+extension&show-deleted-files=true&show-viewed-files=true#diff-be074d694bc3f105d40784fc92a3157bae4200b532472c56282285f9b7a54266R11)